autoscale/src/main/java/com/sequenceiq/periscope/domain/LoadAlert.java

```diff
@@ -0,0 +1,28 @@
+package com.sequenceiq.periscope.domain;
+
+import javax.persistence.DiscriminatorValue;
+import javax.persistence.Entity;
+import javax.persistence.NamedQueries;
+import javax.persistence.NamedQuery;
+import javax.persistence.OneToOne;
+
+@Entity
+@DiscriminatorValue("LOAD")
+@NamedQueries({
+        @NamedQuery(name = "LoadAlert.findByCluster", query = "SELECT c FROM LoadAlert c WHERE c.cluster.id= :clusterId AND c.id= :alertId"),
+        @NamedQuery(name = "LoadAlert.findAllByCluster", query = "SELECT c FROM LoadAlert c WHERE c.cluster.id= :clusterId")
+})
+public class LoadAlert extends BaseAlert {
+
+    @OneToOne
+    private Cluster cluster;
+
```

Comment:
The other fields related to LoadAlert are Min,Max,CoolDown. Since these applies to other type of alerts, I thought it is better to add hostGroup Level config for these three. I can add aggressiveness as another column, think it is better to have column than json, so that we can easily query or update these type of alerts. And I don't think this types of configs can grow to a very large number ..

Reply:
Why I say json is that we don't necessarily know what these fields will be. For the current algo, it's probably aggressiveness as a value between 0 and 1 (fraction / percentage).
If the algorithm changes, the parameters can be very different.

Reply:
ack